### DISCUSSION TITLE
我好奇你知不知道Huggingface?

### DISCUSSION THREAD

**Discussion by huang12zheng** [sean1832/GPT-Brain#15] [General]
虽然我水平一般,但我觉得你可能会觉那个网站很有用
I'm pretty average too, but I thought you might find it useful

**Comment by sean1832** [comment]
哈喽，我最近有点忙。没来得及看。我知道Huggingface，经常在上面玩别人的AI😂。我有时间可以玩玩fine-tuning。但我不懂的是，fine-tune过后的模型难道不是只和我自己的note内容相关吗，所有人都来一次fine-tune难道不会很费时间？而且如果之后增加笔记，不是每次都要重新fine-tune一次？
Hi, I've been busy lately and haven't had a chance to look into it. I know about Huggingface and often play with other people's AI models there 😂. I have will try fine-tuning some time in the future. But I'm confused. Wouldn't the fine-tuned model only be relevant to my own note content? Wouldn't it be time-consuming if everyone had to fine-tune it themselves? And if I add more notes later, would I have to fine-tune it again every time?

**Reply by huang12zheng** [reply]
现在的微调(lora),一次只要2MB左右的显存,一次也只调整对应的部分
不是整个模型
